openflow13/multipart.go

```diff
@@ -958,6 +964,49 @@ func (s *PortStatus) UnmarshalBinary(data []byte) error {
 	return err
 }
 
+///Port Description reply
+type PortDesc struct {
+	ports []*PhyPort
+}
+
+func (s PortDesc) Len() (n uint16) {
+
+	n = 0
+
+	for _, r := range s.ports {
```

Comment:
Can you remove these extra spaces here ?

Reply:
ok，I I adjust the code again